hack/boilerplate/boilerplate.py.txt

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
```

Comment:
do we need this file because we now have the boilerplate.py?

Reply:
Yep, that's why I kept it :)